barbeque/templatetags/barbeque_tags.py

```diff
@@ -24,3 +25,30 @@ def set_tag(context, **kwargs):
 @stringfilter
 def starspan(value):
     return mark_safe(STARSPAN_RE.sub(r'<span>\2</span>', conditional_escape(value)))
+
+
+@register.simple_tag
+def hashed_staticfile(path):
+    try:
+        return staticfiles_storage.hashed_name(path)
+    except (AttributeError, ValueError):
+        return path
+
+
+@register.simple_tag(takes_context=True)
+def page_titleextension(context, page_id, extension):
+    from cms.models import Page
```

Comment:
Can we move the imports to top? (like we did with compressor here: https://github.com/moccu/barbeque/blob/master/barbeque/templatetags/buildcompress.py) - The templatetag should raise a Exception if imports are missing when someone tries to use the tag.